pkg/cli/admin/upgrade/status/mco/layered_node_state.go

```diff
@@ -207,7 +207,9 @@ func isNodeDoneAt(l *LayeredNodeState, pool *mcfgv1.MachineConfigPool) bool {
 }
 
 const (
-	ReasonOfUnavailabilityMCDWorkInProgress = "MCDWorkInProgress"
+	// ReasonOfUnavailabilityMCDWorkInProgress indicates MCD will fix the state and no user intervention is required.
+	ReasonOfUnavailabilityMCDWorkInProgress = "Machine Config Daemon is processing the node"
+	ReasonOfUnavailabilityNodeUnschedulable = "Node is marked unschedulable"
```

Comment:
In different similar assignments of the `ReasonOfUnavailability` field, we start the reason with a lowercase letter `node`. I don't mind either lowercase or uppercase first letter, and I do acknowledge that we don't control everything, as we propagate some information without modification. However, my goal is to try to be as consistent as possible with our messages :raised_hands: 

`Machine Config Daemon is processing the node` is fine as MCD is a name. English is not my first language, but we do refer to Machine Config Daemon with capitalized letters in [documentation](https://docs.openshift.com/container-platform/4.9/nodes/nodes/nodes-nodes-machine-config-daemon-metrics.html).

```suggestion
ReasonOfUnavailabilityNodeUnschedulable = "node is marked unschedulable"
```

Reply:
Let's use uppercase please. I believe Hongkai addresses my [comment](https://github.com/openshift/oc/pull/1838#discussion_r1717210542) here. We used uppercase before #1838.

Reply:
Discussion resolved :100:

Reply:
Although, in the future, we should update the other reasons of unavailibility to be consistent.

Reply:
Yeah, even to the point to uppercasing the first letter of the strings that we take from other sources, such as `failed to drain node`